compiler/ir/ir.tree/gen/org/jetbrains/kotlin/ir/expressions/IrMemberAccessExpression.kt

```diff
@@ -43,11 +43,19 @@ abstract class IrMemberAccessExpression<S : IrSymbol> : IrDeclarationReference()
         return valueArguments[index]
     }
 
+    fun getValueArgumentOrThrow(index: Int): IrExpression {
```

Comment:
I'd prefer to have `getValueArgumentOrThrow` and `getTypeArgumentOrThrow` be declared as extension functions in `IrExpressions.kt` instead of generating them

Reply:
In that case we will be unable to use `valueArguments` and `typeArguments` and create usefull error message

Reply:
Fixed